nodejs/tfwconnector/MessageSender.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+let TFWConnector = require('./TWFConnector');
+
+
+class MessageSender{
+
+    constructor(connector){
+        if(connector instanceof TFWConnector){
+            this.connector = connector;
+        }else{
+            console.error("MessageSender require an instance of TFWConnector!");
+        }
+    }
+
+    send(key, data){
```

Comment:
`MessageSender` should not be used to send data, only instructions (clients should use a naked `TFWConnector` for that (this method should be implemented in `TFWServerConnector`)